synapse/models/files.py

```diff
@@ -556,6 +577,54 @@ def getModelDefs(self):
                     ('base:ext', ('str', {}), {'ro': True,
                         'doc': 'The file extension.'}),
                 )),
+
+                ('file:mime:macho:loadcmd', {}, (
+                    ('file', ('file:bytes', {}), {
+                        'doc': 'The Mach-O file containing the load command.'}),
+                    ('type', ('int', {'enums': s_l_macho.getLoadCmdTypes()}), {
+                        'doc': 'The type of the load command'}),
+                    ('size', ('int', {}), {
+                        'doc': 'The size of the load command structure in bytes.'}),
```

Comment:
Would we want to track the offset of the beginning of the load command so we have sufficient knowledge to use it to slice one out of the file?

Reply:
So, It's fairly trivial to parse out the headers/load commands from a Mach-O (far simpler than an ELF or PE file), so there's not a major performance saving in the future for that.

That and load commands are not set in any particular order other than "my toolchain put them here". All the offsets I see from load commands are absolute file offsets, so it's fairly trivial to relocate load commands relative to each other, so long as the basic structure of <mach header><load commands (and section headers after segment commands)><segments> is obeyed.

So I'm not sure what tracking the offset to the start of the load command structure gets us?